server/controllers/like.controllers.js

```diff
@@ -40,13 +40,13 @@ const toggleLike = async (req, res) => {
     if (userLike) {
       result = await likeHelper.removeLike(user, article);
       return res.status(200).json({
-        message: 'Successfully removed like',
+        message: 'You have successfully disliked an article',
```

Comment:
Since this action is for a particular article and not for any article in general, this can be something like `You have successfully disliked this article`